---
http_interactions:
- request:
    method: post
    uri: http://localhost:3000/api/v1/users
    body:
      encoding: UTF-8
      string: '{"name":"Eric","email":"superuniqueemail@yahoo.com","password":"SuperSecret"}'
    headers:
      User-Agent:
      - Faraday v2.8.1
      Content-Type:
      - application/json
      Accept-Encoding:
      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
      Accept:
      - "*/*"
  response:
    status:
      code: 500
      message: Internal Server Error
    headers:
      Content-Type:
      - application/json; charset=UTF-8
      X-Request-Id:
      - cc268d78-efde-4484-8e2c-7dfde9294435
      X-Runtime:
      - '0.110452'
      Server-Timing:
      - ''
      Content-Length:
      - '18740'
    body:
      encoding: UTF-8
      string: '{"status":500,"error":"Internal Server Error","exception":"#\u003cActiveRecord::NoDatabaseError:\"We
        could not find your database: puppy_play_date_be_development. Available database
        configurations can be found in config/database.yml file.\\n\\nTo resolve this
        error:\\n\\n- Did you create the database for this app, or delete it? You
        may need to create your database.\\n- Has the database name changed? Check
        your database.yml config has the correct database name.\\n\\nTo create your
        database, run:\\n\\n        bin/rails db:create\\n\"\u003e","traces":{"Application
        Trace":[],"Framework Trace":[{"exception_object_id":12480,"id":0,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/postgresql_adapter.rb:74:in
        `rescue in new_client''"},{"exception_object_id":12480,"id":1,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/postgresql_adapter.rb:68:in
        `new_client''"},{"exception_object_id":12480,"id":2,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/postgresql_adapter.rb:980:in
        `connect''"},{"exception_object_id":12480,"id":3,"trace":"activerecord (7.1.2)
        lib/active_record/connection_adapters/postgresql_adapter.rb:992:in `reconnect''"},{"exception_object_id":12480,"id":4,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:685:in `block
        in reconnect!''"},{"exception_object_id":12480,"id":5,"trace":"activesupport
        (7.1.2) lib/active_support/concurrency/null_lock.rb:9:in `synchronize''"},{"exception_object_id":12480,"id":6,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:684:in `reconnect!''"},{"exception_object_id":12480,"id":7,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:788:in `verify!''"},{"exception_object_id":12480,"id":8,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:795:in `connect!''"},{"exception_object_id":12480,"id":9,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:1001:in
        `block in with_raw_connection''"},{"exception_object_id":12480,"id":10,"trace":"activesupport
        (7.1.2) lib/active_support/concurrency/null_lock.rb:9:in `synchronize''"},{"exception_object_id":12480,"id":11,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:1000:in
        `with_raw_connection''"},{"exception_object_id":12480,"id":12,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:1113:in
        `valid_raw_connection''"},{"exception_object_id":12480,"id":13,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/postgresql_adapter.rb:616:in
        `get_database_version''"},{"exception_object_id":12480,"id":14,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/schema_cache.rb:374:in `database_version''"},{"exception_object_id":12480,"id":15,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/schema_cache.rb:70:in `database_version''"},{"exception_object_id":12480,"id":16,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/schema_cache.rb:200:in `database_version''"},{"exception_object_id":12480,"id":17,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:871:in `database_version''"},{"exception_object_id":12480,"id":18,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/postgresql_adapter.rb:644:in
        `check_version''"},{"exception_object_id":12480,"id":19,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract/connection_pool.rb:675:in
        `new_connection''"},{"exception_object_id":12480,"id":20,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract/connection_pool.rb:720:in
        `checkout_new_connection''"},{"exception_object_id":12480,"id":21,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract/connection_pool.rb:699:in
        `try_to_checkout_new_connection''"},{"exception_object_id":12480,"id":22,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract/connection_pool.rb:657:in
        `acquire_connection''"},{"exception_object_id":12480,"id":23,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract/connection_pool.rb:355:in
        `checkout''"},{"exception_object_id":12480,"id":24,"trace":"activerecord (7.1.2)
        lib/active_record/connection_adapters/abstract/connection_pool.rb:184:in `connection''"},{"exception_object_id":12480,"id":25,"trace":"activerecord
        (7.1.2) lib/active_record/migration/pending_migration_connection.rb:8:in `establish_temporary_connection''"},{"exception_object_id":12480,"id":26,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:773:in `block in pending_migrations''"},{"exception_object_id":12480,"id":27,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:772:in `each''"},{"exception_object_id":12480,"id":28,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:772:in `pending_migrations''"},{"exception_object_id":12480,"id":29,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:751:in `check_pending_migrations''"},{"exception_object_id":12480,"id":30,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:643:in `block (2 levels) in call''"},{"exception_object_id":12480,"id":31,"trace":"activesupport
        (7.1.2) lib/active_support/file_update_checker.rb:85:in `execute''"},{"exception_object_id":12480,"id":32,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:648:in `block in call''"},{"exception_object_id":12480,"id":33,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:640:in `synchronize''"},{"exception_object_id":12480,"id":34,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:640:in `call''"},{"exception_object_id":12480,"id":35,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call''"},{"exception_object_id":12480,"id":36,"trace":"activesupport
        (7.1.2) lib/active_support/callbacks.rb:101:in `run_callbacks''"},{"exception_object_id":12480,"id":37,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/callbacks.rb:28:in `call''"},{"exception_object_id":12480,"id":38,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/executor.rb:14:in `call''"},{"exception_object_id":12480,"id":39,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/actionable_exceptions.rb:16:in `call''"},{"exception_object_id":12480,"id":40,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call''"},{"exception_object_id":12480,"id":41,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call''"},{"exception_object_id":12480,"id":42,"trace":"railties
        (7.1.2) lib/rails/rack/logger.rb:37:in `call_app''"},{"exception_object_id":12480,"id":43,"trace":"railties
        (7.1.2) lib/rails/rack/logger.rb:24:in `block in call''"},{"exception_object_id":12480,"id":44,"trace":"activesupport
        (7.1.2) lib/active_support/tagged_logging.rb:135:in `block in tagged''"},{"exception_object_id":12480,"id":45,"trace":"activesupport
        (7.1.2) lib/active_support/tagged_logging.rb:39:in `tagged''"},{"exception_object_id":12480,"id":46,"trace":"activesupport
        (7.1.2) lib/active_support/tagged_logging.rb:135:in `tagged''"},{"exception_object_id":12480,"id":47,"trace":"activesupport
        (7.1.2) lib/active_support/broadcast_logger.rb:240:in `method_missing''"},{"exception_object_id":12480,"id":48,"trace":"railties
        (7.1.2) lib/rails/rack/logger.rb:24:in `call''"},{"exception_object_id":12480,"id":49,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/remote_ip.rb:92:in `call''"},{"exception_object_id":12480,"id":50,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/request_id.rb:28:in `call''"},{"exception_object_id":12480,"id":51,"trace":"rack
        (3.0.8) lib/rack/runtime.rb:24:in `call''"},{"exception_object_id":12480,"id":52,"trace":"activesupport
        (7.1.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in
        `call''"},{"exception_object_id":12480,"id":53,"trace":"actionpack (7.1.2)
        lib/action_dispatch/middleware/server_timing.rb:59:in `block in call''"},{"exception_object_id":12480,"id":54,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/server_timing.rb:24:in `collect_events''"},{"exception_object_id":12480,"id":55,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/server_timing.rb:58:in `call''"},{"exception_object_id":12480,"id":56,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/executor.rb:14:in `call''"},{"exception_object_id":12480,"id":57,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/static.rb:25:in `call''"},{"exception_object_id":12480,"id":58,"trace":"rack
        (3.0.8) lib/rack/sendfile.rb:114:in `call''"},{"exception_object_id":12480,"id":59,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/host_authorization.rb:141:in `call''"},{"exception_object_id":12480,"id":60,"trace":"railties
        (7.1.2) lib/rails/engine.rb:529:in `call''"},{"exception_object_id":12480,"id":61,"trace":"puma
        (6.4.0) lib/puma/configuration.rb:272:in `call''"},{"exception_object_id":12480,"id":62,"trace":"puma
        (6.4.0) lib/puma/request.rb:100:in `block in handle_request''"},{"exception_object_id":12480,"id":63,"trace":"puma
        (6.4.0) lib/puma/thread_pool.rb:378:in `with_force_shutdown''"},{"exception_object_id":12480,"id":64,"trace":"puma
        (6.4.0) lib/puma/request.rb:99:in `handle_request''"},{"exception_object_id":12480,"id":65,"trace":"puma
        (6.4.0) lib/puma/server.rb:443:in `process_client''"},{"exception_object_id":12480,"id":66,"trace":"puma
        (6.4.0) lib/puma/server.rb:241:in `block in run''"},{"exception_object_id":12480,"id":67,"trace":"puma
        (6.4.0) lib/puma/thread_pool.rb:155:in `block in spawn_thread''"}],"Full Trace":[{"exception_object_id":12480,"id":0,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/postgresql_adapter.rb:74:in
        `rescue in new_client''"},{"exception_object_id":12480,"id":1,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/postgresql_adapter.rb:68:in
        `new_client''"},{"exception_object_id":12480,"id":2,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/postgresql_adapter.rb:980:in
        `connect''"},{"exception_object_id":12480,"id":3,"trace":"activerecord (7.1.2)
        lib/active_record/connection_adapters/postgresql_adapter.rb:992:in `reconnect''"},{"exception_object_id":12480,"id":4,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:685:in `block
        in reconnect!''"},{"exception_object_id":12480,"id":5,"trace":"activesupport
        (7.1.2) lib/active_support/concurrency/null_lock.rb:9:in `synchronize''"},{"exception_object_id":12480,"id":6,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:684:in `reconnect!''"},{"exception_object_id":12480,"id":7,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:788:in `verify!''"},{"exception_object_id":12480,"id":8,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:795:in `connect!''"},{"exception_object_id":12480,"id":9,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:1001:in
        `block in with_raw_connection''"},{"exception_object_id":12480,"id":10,"trace":"activesupport
        (7.1.2) lib/active_support/concurrency/null_lock.rb:9:in `synchronize''"},{"exception_object_id":12480,"id":11,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:1000:in
        `with_raw_connection''"},{"exception_object_id":12480,"id":12,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:1113:in
        `valid_raw_connection''"},{"exception_object_id":12480,"id":13,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/postgresql_adapter.rb:616:in
        `get_database_version''"},{"exception_object_id":12480,"id":14,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/schema_cache.rb:374:in `database_version''"},{"exception_object_id":12480,"id":15,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/schema_cache.rb:70:in `database_version''"},{"exception_object_id":12480,"id":16,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/schema_cache.rb:200:in `database_version''"},{"exception_object_id":12480,"id":17,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract_adapter.rb:871:in `database_version''"},{"exception_object_id":12480,"id":18,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/postgresql_adapter.rb:644:in
        `check_version''"},{"exception_object_id":12480,"id":19,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract/connection_pool.rb:675:in
        `new_connection''"},{"exception_object_id":12480,"id":20,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract/connection_pool.rb:720:in
        `checkout_new_connection''"},{"exception_object_id":12480,"id":21,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract/connection_pool.rb:699:in
        `try_to_checkout_new_connection''"},{"exception_object_id":12480,"id":22,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract/connection_pool.rb:657:in
        `acquire_connection''"},{"exception_object_id":12480,"id":23,"trace":"activerecord
        (7.1.2) lib/active_record/connection_adapters/abstract/connection_pool.rb:355:in
        `checkout''"},{"exception_object_id":12480,"id":24,"trace":"activerecord (7.1.2)
        lib/active_record/connection_adapters/abstract/connection_pool.rb:184:in `connection''"},{"exception_object_id":12480,"id":25,"trace":"activerecord
        (7.1.2) lib/active_record/migration/pending_migration_connection.rb:8:in `establish_temporary_connection''"},{"exception_object_id":12480,"id":26,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:773:in `block in pending_migrations''"},{"exception_object_id":12480,"id":27,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:772:in `each''"},{"exception_object_id":12480,"id":28,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:772:in `pending_migrations''"},{"exception_object_id":12480,"id":29,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:751:in `check_pending_migrations''"},{"exception_object_id":12480,"id":30,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:643:in `block (2 levels) in call''"},{"exception_object_id":12480,"id":31,"trace":"activesupport
        (7.1.2) lib/active_support/file_update_checker.rb:85:in `execute''"},{"exception_object_id":12480,"id":32,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:648:in `block in call''"},{"exception_object_id":12480,"id":33,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:640:in `synchronize''"},{"exception_object_id":12480,"id":34,"trace":"activerecord
        (7.1.2) lib/active_record/migration.rb:640:in `call''"},{"exception_object_id":12480,"id":35,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/callbacks.rb:29:in `block in call''"},{"exception_object_id":12480,"id":36,"trace":"activesupport
        (7.1.2) lib/active_support/callbacks.rb:101:in `run_callbacks''"},{"exception_object_id":12480,"id":37,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/callbacks.rb:28:in `call''"},{"exception_object_id":12480,"id":38,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/executor.rb:14:in `call''"},{"exception_object_id":12480,"id":39,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/actionable_exceptions.rb:16:in `call''"},{"exception_object_id":12480,"id":40,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call''"},{"exception_object_id":12480,"id":41,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call''"},{"exception_object_id":12480,"id":42,"trace":"railties
        (7.1.2) lib/rails/rack/logger.rb:37:in `call_app''"},{"exception_object_id":12480,"id":43,"trace":"railties
        (7.1.2) lib/rails/rack/logger.rb:24:in `block in call''"},{"exception_object_id":12480,"id":44,"trace":"activesupport
        (7.1.2) lib/active_support/tagged_logging.rb:135:in `block in tagged''"},{"exception_object_id":12480,"id":45,"trace":"activesupport
        (7.1.2) lib/active_support/tagged_logging.rb:39:in `tagged''"},{"exception_object_id":12480,"id":46,"trace":"activesupport
        (7.1.2) lib/active_support/tagged_logging.rb:135:in `tagged''"},{"exception_object_id":12480,"id":47,"trace":"activesupport
        (7.1.2) lib/active_support/broadcast_logger.rb:240:in `method_missing''"},{"exception_object_id":12480,"id":48,"trace":"railties
        (7.1.2) lib/rails/rack/logger.rb:24:in `call''"},{"exception_object_id":12480,"id":49,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/remote_ip.rb:92:in `call''"},{"exception_object_id":12480,"id":50,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/request_id.rb:28:in `call''"},{"exception_object_id":12480,"id":51,"trace":"rack
        (3.0.8) lib/rack/runtime.rb:24:in `call''"},{"exception_object_id":12480,"id":52,"trace":"activesupport
        (7.1.2) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in
        `call''"},{"exception_object_id":12480,"id":53,"trace":"actionpack (7.1.2)
        lib/action_dispatch/middleware/server_timing.rb:59:in `block in call''"},{"exception_object_id":12480,"id":54,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/server_timing.rb:24:in `collect_events''"},{"exception_object_id":12480,"id":55,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/server_timing.rb:58:in `call''"},{"exception_object_id":12480,"id":56,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/executor.rb:14:in `call''"},{"exception_object_id":12480,"id":57,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/static.rb:25:in `call''"},{"exception_object_id":12480,"id":58,"trace":"rack
        (3.0.8) lib/rack/sendfile.rb:114:in `call''"},{"exception_object_id":12480,"id":59,"trace":"actionpack
        (7.1.2) lib/action_dispatch/middleware/host_authorization.rb:141:in `call''"},{"exception_object_id":12480,"id":60,"trace":"railties
        (7.1.2) lib/rails/engine.rb:529:in `call''"},{"exception_object_id":12480,"id":61,"trace":"puma
        (6.4.0) lib/puma/configuration.rb:272:in `call''"},{"exception_object_id":12480,"id":62,"trace":"puma
        (6.4.0) lib/puma/request.rb:100:in `block in handle_request''"},{"exception_object_id":12480,"id":63,"trace":"puma
        (6.4.0) lib/puma/thread_pool.rb:378:in `with_force_shutdown''"},{"exception_object_id":12480,"id":64,"trace":"puma
        (6.4.0) lib/puma/request.rb:99:in `handle_request''"},{"exception_object_id":12480,"id":65,"trace":"puma
        (6.4.0) lib/puma/server.rb:443:in `process_client''"},{"exception_object_id":12480,"id":66,"trace":"puma
        (6.4.0) lib/puma/server.rb:241:in `block in run''"},{"exception_object_id":12480,"id":67,"trace":"puma
        (6.4.0) lib/puma/thread_pool.rb:155:in `block in spawn_thread''"}]}}'
  recorded_at: Sat, 06 Jan 2024 23:26:53 GMT
recorded_with: VCR 6.2.0
